blkn/src/Input/index.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import * as React from "react";
+import Typography from "./../Typography";
+
+type Props = {
+  label?: string,
+  error?: string,
+  value?: string,
+  onChange?: (SyntethicInputEvent<HTMLInputElement>) => any,
+};
+
+const Input = (props: Props) => (
+  <div className="root">
+    <label>
+      {props.label && (
+        <div>
+          <Typography type="secondary">{props.label}</Typography>
+        </div>
+      )}
+      <input value={props.value} onChange={props.onChange} />
+    </label>
+    {props.error && (
+      <div>
+        <Typography size="small" type="error">
+          {props.error}
+        </Typography>
+      </div>
+    )}
+    <style jsx>{`
```

Comment:
shouldn't some of the values be somehow centralized, such as `font-size`, colors, `line-height` etc?

Reply:
that's definitely worth a thought, I'm just not decided yet

Reply:
Will be optimized when it's needed - with more components.